src/Xablu.WebApiClient.sln

```diff
@@ -130,6 +131,7 @@ Global
 		{69AFB91C-FF88-45AD-BCCF-9D1F3B428111}.Release|x64.Build.0 = Release|Any CPU
 		{69AFB91C-FF88-45AD-BCCF-9D1F3B428111}.Release|x86.ActiveCfg = Release|Any CPU
 		{69AFB91C-FF88-45AD-BCCF-9D1F3B428111}.Release|x86.Build.0 = Release|Any CPU
+<<<<<<< HEAD
```

Comment:
This is an unresolved merge conflict, can you please check this?

Reply:
This is baffling me, i had no more conflicts when pushing 🤔.